projects/arlas-components/assets/i18n/fr.json

```diff
@@ -111,5 +111,15 @@
 	"Cloud": "Nuage",
 	"country": "Pays",
 	"None": "Aucune",
-	"Remove this selection":"Supprimer cette sélection"
-}
\ No newline at end of file
+	"Remove this selection":"Supprimer cette sélection",
+  "legend": "{{metric}} des {{field}} ({{normalized}})",
+  "legend without normalized": "{{metric}} des {{field}}",
+  "legend without field": "{{metric}} ({{normalized}})",
```

Comment:
isn't it always the count that does not have a field ?

Reply:
Yes your right , but I think it could be better to keep something more generic.